consensus/module.go

```diff
@@ -1,37 +1,44 @@
 package consensus
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
 
 	"github.com/pokt-network/pocket/consensus/leader_election"
-	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
 	typesCons "github.com/pokt-network/pocket/consensus/types"
 	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
-	"github.com/pokt-network/pocket/shared/modules"
-	"github.com/pokt-network/pocket/shared/types"
-	"github.com/pokt-network/pocket/shared/types/genesis"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
+
+	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
+	"github.com/pokt-network/pocket/shared/modules"
 )
 
 const (
-	DefaultLogPrefix string = "NODE" // Just a default that'll be replaced during consensus operations.
+	DefaultLogPrefix    string = "NODE" // Just a default that'll be replaced during consensus operations.
+	ConsensusModuleName        = "consensus"
```

Comment:
The goal of creating constants is to be able to easily use the same variable in different / multiple locations, but this is only being used in one place as far as I can tell. Why not simply do this:

```
func (m *ConsensusModule) GetModuleName() string {
	return "consensus"
}
```

Reply:
Makes sense.